umap/models.py

```diff
@@ -436,6 +440,7 @@ def metadata(self, user=None, request=None):
 
     def clone(self, map_inst=None):
         new = self.__class__.objects.get(pk=self.pk)
+        new._state.adding = True
```

Comment:
This is [recommended by Django when cloning objects](https://docs.djangoproject.com/en/5.0/topics/db/queries/#copying-model-instances), to have a way to mark the object as "to be saved". (you can see it as `isDirty` for Django models).

> Model._state[¶](https://docs.djangoproject.com/en/5.0/ref/models/instances/#django.db.models.Model._state)
> 
>    The _state attribute refers to a ModelState object that tracks the lifecycle of the model instance.
>    The ModelState object has two attributes: adding, a flag which is True if the model has not been saved to the database yet, and db, a string referring to the database alias the instance was loaded from or saved to.
> 
>    Newly instantiated instances have adding=True and db=None, since they are yet to be saved. Instances fetched from a QuerySet will have adding=False and db set to the alias of the associated database.